canvas_modules/common-canvas/src/common-properties/components/editor-form/editor-form.scss

```diff
@@ -124,7 +148,7 @@ $primary-tab-height: $spacing-08;
 }
 
 // Display in full height if inside a tearsheet
-.properties-editor-form.tearsheet-container {
+.properties-editor-form.tearsheet-container, .properties-editor-form.right-flyout-tabs-view {
```

Comment:
Can't scroll because of `overflow: hidden`.
<img width="635" alt="Screenshot 2023-05-24 at 5 07 26 PM" src="https://github.com/elyra-ai/canvas/assets/25124000/a0b9a97f-89b1-4680-8c1c-f52467a31a52">